### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by nassiba-chiyahou** [community/community#145109] [GitHub Education]
### Hi everyone,

Hello GitHub Community!

I am writing about GitHub Student Developer Pack application. my request was approved 5 days ago and I have not received any of the Student pack benefits. I wish if someone could help me with this issue from GitHub Education Team.
![image](https://github.com/user-attachments/assets/ffbffc6a-c608-4310-be81-c7273542aede)

Thanks for your time and support in advance :)

**Comment by JoaoCoelho2003** [comment]
If your benefits for the GitHub Student Developer Pack have not yet been activated you can start by contacting GitHub Support. They can provide specific details about your application status and assist in resolving any issues. You can submit a request through the GitHub Support Page.

Additionally, ensure you meet all the requirements for the Student Developer Pack, such as providing the necessary documentation to verify your student status. Keep in mind that benefits are usually activated within 72 hours of approval, but during peak times, like the start of a new semester, the process may take longer ( in my opinion this is most likely the problem that you are facing, it might be a good idea to just wait a little bit more).

It may also be worthwhile to check your email to see if GitHub has contacted you regarding your request. 

If there is still a delay, it might be worth following up with your educational institution to ensure they have completed any required steps for the verification process.

**Comment by davevad93** [comment]
Hi @nassiba-chiyahou ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @nassiba-chiyahou was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
